### PR TITLE
added details on calendar

### PR DESCRIPTION
**Pull request by weining-li**
i wasn't sure how to get the IDs from database so that wasn't included

### CODE REVIEW

File: src/components/TaskPreview.js [hunk 1]
```diff
@@ -50,10 +67,114 @@ function EventReview(props) {
         <Typography variant="body2" color="textSecondary" component="p">
           Hello the world. Just do something.... What... Exercise is good for your health
```
Comment (allygator) [inline src/components/TaskPreview.js]: Can we fix this block to not have this place holder text?

**Review by allygator — CHANGES_REQUESTED** [review]

File: src/components/TaskPreview.js (hunk 1 above)
Comment (weining-li) [reply src/components/TaskPreview.js]: yas

Comment (allygator) [reply src/components/TaskPreview.js]: There needs to be a bit of placeholder text, "Expand for details" or something? maybe?

Comment (allygator) [reply src/components/TaskPreview.js]: I take that back, its fine without for now.

**Review by allygator — APPROVED** [review]

**Pull request merged**